### PR TITLE
[FIX JENKINS-47248] Add warning / message that saving Jenkinsfiles without SSH is unsupported (1.3 branch)

### PR DESCRIPTION
**Pull request by kzantow**
# Description

Tweak to add text indicating saving using non-SSH is not supported. Needs more tests in this area.

image

image

See [JENKINS-47248](https://issues.jenkins-ci.org/browse/JENKINS-47248).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by michaelneale** [comment]
look ok @i386 ?

**Comment by i386** [comment]
LGTM 🚢

**Comment by michaelneale** [comment]
can update this with release/1.3 tomorrow (once there is a pending change to fix the ATH problem there)

File: blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/Property.java [hunk 1]
```diff
@@ -133,24 +133,21 @@ public void writeTo(Object object, TreePruner pruner, DataWriter writer) throws
         TreePruner child = pruner.accept(object, this);
         if (child==null)        return;
 
-        Object d = writer.getExportConfig().getExportInterceptor().getValue(this,object,writer.getExportConfig());
+        Object d = writer.getExportConfig().getExportInterceptor().getValue(this,object, writer.getExportConfig());
```
Comment (vivek) [inline blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/Property.java]: Why are these changes appearing over here? PR for stapler related changes is already merged.

**Comment by kzantow** [comment]
@vivek botched merge, had this branch open in 2 PRs and confused myself; fixed

**Comment by vivek** [comment]
Thanks @kzantow!

**Comment by michaelneale** [comment]
LGTM - keep in mind @kzantow that 1.3.0 is out - so this would be in a 1.3.1 if/when it goes out

**Pull request merged**